### ISSUE TITLE
Docs website seems to be down

### ISSUE THREAD

**Issue by ramon-san** [JustFly1984/react-google-maps-api#3344]
Simple issue, the docs website seems to be down. I'm trying to access from China so this sometimes poses problems, already tried different means of access and none seem to be working. Just want to confirm if this is happening to more people.

https://react-google-maps-api-docs.netlify.app/

<img width="541" alt="image" src="https://github.com/JustFly1984/react-google-maps-api/assets/65260209/e381ce85-a4c6-4c84-b756-b6b73e053e0e">

**Comment by 0XYoussefX0** [comment]
Yes, the website is down for everyone. You can access the docs from [Web Archive](https://web.archive.org/web/20230701010714mp_/https://react-google-maps-api-docs.netlify.app/) or [Tom Chen's Repo](https://github.com/tomchentw/react-google-maps)

**Comment by DrFrancisco222** [comment]
Is there a solution for this? I can't find any documentation to use Autocomplete

**Comment by JustFly1984** [comment]
I've pulled it down. Need to fix docs config to deploy it again.

**Comment by brunocoelho** [comment]
Thanks for the hard work on putting it back on! 🚀 This repo is extremely important for a lot of projects.

**Comment by JustFly1984** [comment]
Docs are here in the repo in packages/react-google-maps-api/ there is markdown files. Also code examples in packages/react-google-maps-gatsby-example

**Comment by DariuszLegizynski** [comment]
Duplicate #3325

**Comment by dmurawsky** [comment]
bump
